table-api/src/main/java/io/deephaven/api/agg/Count.java

```diff
@@ -0,0 +1,31 @@
+package io.deephaven.api.agg;
+
+import io.deephaven.api.ColumnName;
+import io.deephaven.api.JoinMatch;
+import org.immutables.value.Value.Immutable;
+import org.immutables.value.Value.Parameter;
+
+@Immutable(builder = false, copy = false)
+public abstract class Count implements Aggregation {
+
+    public static Count of(ColumnName name) {
+        return ImmutableCount.of(name);
+    }
+
+    public static Count of(String x) {
+        return of(ColumnName.of(x));
+    }
+
+    @Parameter
+    public abstract ColumnName column();
+
+    public final JoinMatch match() {
```

Comment:
`@Override`.

Reply:
This isn't really a `JoinMatch`. Arguably, none of the aggs have a join match. They have a `ColumnPair` where one is from the input and one is in the output, and in the `Count` case there's no input column.

Reply:
This has been updated w/ an agg specific pair type.